dpctl/tests/test_sycl_device_factory.py

```diff
@@ -51,8 +51,6 @@ def string_to_device_type(dty_str):
         return dty.accelerator
     elif dty_str == "cpu":
         return dty.cpu
```

Comment:
The `"host"` device type is still present in `docs/_legacy/docfiles/user_guides/manual/dpctl/device_selection.rst`. Should it be cleaned up there?

Also, I note that `device_selection.rst` page is not available on `https://intelpython.github.io/dpctl/latest/index.html`. Is that expected?

Reply:
Both cases are expected and fine. The `_legacy/*` files are the old documentation, from prior to the overhaul in https://github.com/IntelPython/dpctl/pull/1619